### ISSUE TITLE
Layout improvement

### ISSUE THREAD

**Issue by Amzani** [asyncapi/studio#703]
## Problem

> What's the story exposing the problem the users are facing

The previous top bar disappears as it was useless (it only had the Studio logo and some links to stuff not related to Studio). Instead, it's now holding the name of the service (AsyncAPI file) we're developing and a few buttons to customize the layout. These are just the buttons that used to be on the sidebar, i.e., toggle Navigation, Docs, Diagnostics, etc.

But probably, the biggest change here is the introduction of this new sidebar on the left. It contains a bunch of buttons and also the avatar of the user. This new sidebar is meant to be the navigation for the whole application, meaning it's not just for the code editor as it was before. The purpose of this sidebar is to navigate to other parts of the application like Services, Governance, Settings, and more.

In general, now the layout is accommodated to start growing the application in a friendly and sane way.

## Solution

> Proposed solution with mockups, views ...

![Image](https://user-images.githubusercontent.com/554438/242863467-4e436934-1a1d-44e4-8e56-6cf824ab7e61.png)

Link to Figma: https://www.figma.com/proto/bPMB3lkMTOOMuKk0oGLuNm/Studio?type=design&node-id=144-4141&scaling=contain&page-id=0%3A1&starting-point-node-id=2%3A2

## Rabbit holes

> Potential technical, design ... challenges

1. To incorporate this change, we plan to leverage the new design system (refer to issue #636) as our foundation. Before adding the components into studio, it's crucial to ensure they are first integrated into the design system, in case they are not already included.

## Scope

> Describe a list of tasks or issues that needs to be done ( you don't need to have an exhaustive list of all the tasks in the beginning) 

- [ ] Issue 1
- [ ] Issue 2

## Out of bounds

> What won't be part of the scope

## Success criteria

> How do we know we made a good bet

**Comment by divyammadhok** [comment]
Hey @Amzani Let me know if you require assistance in solving this issue. I can pick it up if the issue is unblocked. I'll go through the issue and clarify any requirements once I get the go-ahead from your end.

**Comment by Amzani** [comment]
Hi @divyammadhok This work will start after cycle 1 is done: https://shapeit.app/projects/org/asyncapi/16/cycles/e4232524
I'll ping you here so you can contribute once we start the cycle 2.

**Comment by divyammadhok** [comment]
Thanks @Amzani Sounds good!

**Comment by Amzani** [comment]
Do we want to wait until the next architecture https://github.com/asyncapi/studio/issues/725 is completed before integrating the components ?

Any thoughts ? @Shurtu-gal @fmvilas @KhudaDad414

**Comment by Shurtu-gal** [comment]
Yeah we wouldn't want to increase the task of migration by adding components before everything is decided.

**Comment by divyammadhok** [comment]
@Amzani Can you share the access to this channel

**Comment by fmvilas** [comment]
@divyammadhok sorry for the silence. There have been summer holidays recently and @Amzani probably missed your message. You can sign in to our Slack here: https://asyncapi.com/slack-invite. After that, you'll be able to see the content of this link. Hope that helps.

**Comment by divyammadhok** [comment]
Hey @fmvilas Got through your message now, and I believe that the link is not valid anymore. Can you please ping this to me again? I'll make a note of joining it on time.

**Comment by Min2who** [comment]
Hey @Amzani can i give a hand or two in this?

**Comment by utnim2** [comment]
Hii @Shurtu-gal  @Amzani @KhudaDad414 
![image](https://github.com/asyncapi/studio/assets/127925465/73ce69e4-c8ad-4b2d-bc8a-566945a19152)

Is there been any plan on how to show the user avatar dynamically?

**Comment by Amzani** [comment]
We don't need to add the avatar as we don't support users (signin/signup) in studio @utnim2

**Comment by utnim2** [comment]
> We don't need to add the avatar as we don't support users (signin/signup) in studio @utnim2 

@Amzani many parts of the studio are not completed yet that us there in the figma file 
should i proceed with the part that is present ?

Thanks

**Comment by Amzani** [comment]
Best to coordinate with @Shurtu-gal as he is working on the NextJS migration, I suggest to add this enhancement as part of the new app in `apps/studio-next`

**Comment by Amzani** [comment]
@utnim2 @Shurtu-gal is working on https://github.com/asyncapi/studio/pull/997 (still under review), you will need this.

**Comment by utnim2** [comment]
i am working on sidebar and the topbar

**Comment by ibishal** [comment]
We need to add the `topbar` and the `sidebar` as the design provide right?

cc @Amzani @Shurtu-gal 

i am taking this, if no one is working in it
